pages/Configuring/Uncommon-tips-&-tricks.md

```diff
@@ -114,6 +114,66 @@ bind = $mod, S, movetoworkspace, special:magic
 bind = $mod, S, togglespecialworkspace, magic
 ```
 
+## Show desktop
+
+This approach uses same principle as the `Minimize windows using special workspaces` section.
+It moves all windows from current workspace to a `special` workspace named `desktop`.
+Showing desktop state is remembered per workspace.
+
+Create a script:
+```sh
+#!/usr/bin/env sh
+
+TMP_FILE="/tmp/hyprland-show-desktop"
+
+NUMBER_HIDDEN_WINDOWS=$(hyprctl workspaces -j | jq '.[] | select (.name=="special:desktop") | .windows')
+
+CW=$(hyprctl monitors -j | jq '.[] | .activeWorkspace | .name' | sed 's/"//g')
+
+echo $CW
+echo $NUMBER_HIDDEN_WINDOWS
+
+if [ -s "$TMP_FILE-$CW" ]; then
+  readarray -d $'\n' -t AA <<< $(< "$TMP_FILE-$CW")
+
+  for address in "${AA[@]}"
+  do
+    echo "HERE"
+    cmd="hyprctl dispatch movetoworkspacesilent name:$CW,address:$address"
+    eval $cmd
```

Comment:
```suggestion
    hyprctl dispatch movetoworkspacesilent "name:$CW,address:$address"
```

leftover debug statements?

Also, instead of spamming the socket you could append commands to a variable and then do `hyprctl --batch $COMMANDS`.

Reply:
``` hyprctl dispatch movetoworkspacesilent "name:$CW,address:$address" ``` doesn't work for me i tried it $address turns out empty for some reason